code/edgeidentity/build.gradle.kts

```diff
@@ -41,7 +39,7 @@ dependencies {
     // ANDROIDX_TEST_EXT_JUNIT, ESPRESSO_CORE
 
     androidTestImplementation ("com.fasterxml.jackson.core:jackson-databind:2.12.7")
-    androidTestImplementation("com.adobe.marketing.mobile:identity:$functionalTestIdentityVersion-SNAPSHOT")
+    androidTestImplementation("com.adobe.marketing.mobile:identity:$functionalTestIdentityVersion")
```

Comment:
Do we still need to exclude core here?

Reply:
No, identity is bundled with Core and they are set to the same version.

Reply:
let's update he lines below 👇